week-3-practise/level-3/1-practise-all.js

```diff
@@ -45,4 +47,14 @@ PasswordValidationResult=  [false, false, false, false, true]
 
 var password = ["Se%5", "TktE.TJTU", "384HsHF", "dvyyeyY!5", "tryT3729."];
 
-console.log(PasswordValidationResult)
\ No newline at end of file
+var PasswordValidationResult = password.map(passkey =>
+  passkey.length >= 5 &&
+  passkey.match(/[A-Z]/g) &&
+  passkey.match(/[a-z]/g) &&
+  passkey.match(/[0-9]/g) &&
+  passkey !== password
+    ? true
```

Comment:
You can simply use `passkey !== password`. There is no need for `? true : false`